docker-compose.yml

```diff
@@ -1,17 +1,19 @@
 version: '2'
 services: 
-    xnat-web:
-       build: ./tomcat
+   xnat-web:
+       build: ./xnat17
        ports:
-        - "8081:8080"
+         - "8081:8080"
        volumes:
-         - ./webapps:/opt/tomcat/webapps
-         - ./plugins:/data/xnat/home/plugins
-         - /var/run/docker.sock:/var/run/docker.sock
-       expose:
-        - "8080"
+         - ./xnat-home:/data/xnat
```

Comment:
Shouldn't this mount /data/xnat/home?

Reply:
I have near zero knowledge on how xnat stores data so I just exposed the whole /data/xnat as a volume. Now that you mention it, ./xnat-home is a bad name indeed.

Should we think about what data needs to be exposed as volumes and what shouldn't since containers should be non-persistent? Or did someone already figure this out?

Reply:
That's an interesting question, but perhaps out of scope for this PR. I would think just leave this mostly as you have it now, but change the mount to `./xnat-home:/data/xnat/home`.